### ISSUE TITLE
Consider adding Fable support to Expo

### ISSUE THREAD

**Issue by et1975** [fable-compiler/fable-react-native#16]
Seems RN's Getting Started now [recommends Expo](https://facebook.github.io/react-native/docs/getting-started.html#testing-your-react-native-installation), would be nice to see Fable option there, as they already have [ClojureScript section](https://docs.expo.io/versions/latest/guides/using-clojurescript.html).

**Comment by alfonsogarciacaro** [comment]
Yes, I saw it a while ago. I haven't used Expo myself yet, but it seems to ease a lot the setup for mobile development so it would be very useful for newcomers. I'm not sure if I'd be able to do it alone but I could help someone who wants to integrate Fable with Expo, is someone interested? @forki @Krzysztof-Cieslak

**Comment by forki** [comment]
Expo is VERY limited. I personally don't see any usecase other than hello
world

Am 22.11.2017 09:23 schrieb "Alfonso Garcia-Caro" <notifications@github.com
